pygmt/tests/test_ternary.py

```diff
@@ -0,0 +1,23 @@
+"""
+Tests for pygmt.ternary.
+"""
+import pytest
+from pygmt import Figure, makecpt
+
+
+@pytest.mark.mpl_image_compare
+def test_ternary():
+    """
+    Test plotting a ternary chart.
+    """
+    fig = Figure()
+    makecpt(cmap="turbo", series=[0, 80, 10])
+    fig.ternary(
+        table="@ternary.txt",
+        region="0/100/0/100/0/100",
+        cmap=True,
```

Comment:
Is it possible to modify the test so that it doesn't require `makecpt`? Just want this to be a proper 'unit' test that tests the `ternary` function only.

Reply:
I'll see if I can use a color pallete without `makecpt`; I've only tried it once but was unsuccessful.

Reply:
It's not the best graph, but I changed the test to use `cmap="red,orange,yellow,green,blue,violet",`. It doesn't apply color to all of the plotted data points, but I want to avoid an exceptionally long string for the CPT.